storage/rocksdb/rdb_datadic.cc

```diff
@@ -3459,6 +3460,20 @@ void Rdb_tbl_def::set_name(const std::string &name) {
   check_if_is_mysql_system_table();
 }
 
+std::shared_ptr<Rdb_key_def> Rdb_tbl_def::get_pk() {
```

Comment:
We issue get_pk() mainly to pull out the gl_index_id() that is used for the auto_increment key. Can we be more specific here for this function? Call it get_autoincr_gl_index_id() perhaps, so it's clear it's generating the index used for storing the auto_increment key. This also allows us to skip copying the Rdb_key_def share ptr.